bin/cli.sh

```diff
@@ -7,13 +7,21 @@ PROJ_ROOT=${THIS_DIR}/..
 
 pushd ${PROJ_ROOT} > /dev/null
 
-# Default CLI image
-VERSION=$(cat VERSION)
-DEFAULT_IMAGE=faasm/cli:${VERSION}
+# See if someone has specified a CLI image through an environment variable
+if [[ -z "${CLI_IMAGE}" ]]; then
+    # Prepare the default version
+    VERSION=$(cat VERSION)
+    DEFAULT_IMAGE=faasm/cli:${VERSION}
+
+    # Take the script argument if provided, else use the default
+    export CLI_IMAGE=${1:-${DEFAULT_IMAGE}}
+fi
+
+echo "Running Faasm CLI (${CLI_IMAGE})"
+
```

Comment:
As usual I've slipped in a small improvement unrelated to the PR title :smile: 

This lets a user set an environment variable for their CLI container, as I got tired of adding my own image name when calling `./bin/cli.sh`.

Happy to shift to another PR if not clear.

Reply:
no probs. this makes sense, I actually have this on my aliases:
```
alias workon-faasm="cd /home/csegarra/faasm/; ./bin/cli.sh csg-workon/faasm:0.1"
```